Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -964,6 +964,12 @@ void TextEditor::keydown_event(KeyEvent& event)
                 auto word_break_pos = document().first_word_break_after(m_cursor);
                 erase_count = word_break_pos.column() - m_cursor.column();
             }
+
+            auto const new_cursor = TextPosition(m_cursor.line(), m_cursor.column());
+
+            // If we are not deleting a selection, then lets check if there is an emoji or glyph with multiple code points.
+            erase_count = document().get_code_points_after_cursor(new_cursor);
```

Comment:
This will always override the `ctrl` case? let's either put this in an `else` or handle `ctrl`, but we definitely can't ignore that modifier